.ci/deps.opam.sh

```diff
@@ -11,6 +11,7 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
   opam init --y
   opam update
   opam pin add --yes --no-action atdgen 1.10.0
+  opam pin add --yes --no-action easy-format 1.2.0
```

Comment:
a comment about that would be good IMO so it can be removed when infer fixes that and people know what this is about

Reply:
infer is at version 0.13.0
They have fixed it.
This is yet another patch to keep alive a horribly old version.